auth/src/main/java/com/firebase/ui/auth/ui/account_link/SaveCredentialsActivity.java

```diff
@@ -89,7 +89,7 @@ protected void onStart() {
 
     @Override
     public void onConnected(@Nullable Bundle bundle) {
-        if (mEmail == null) {
+        if (mEmail == null || mEmail.equals("")) {
```

Comment:
Empty email issue happens due to privacy setting of some users, where you get name and profile image but email is null